### DISCUSSION TITLE
Error instaling :(

### DISCUSSION THREAD

**Discussion by soyatlas** [Mikubill/sd-webui-controlnet#1023] [Q&A]
Hello, I am trying install controlnet in A1111
I had this configuration. 
A clean installation Python 3.10.6 as indicate A1111 + Git
computer:
AMD CPU (9 5900)
128 RAM
GPU 3090 24ram

running A1111 the info is:
python: 3.10.6  •  torch: 1.13.1+cu117  •  xformers: N/A  •  gradio: 3.23.0  

I had a clean AUTOMATIC1111/stable-diffusion-webui/
no more extensions, a webui-user.bat as original.

1-sorry my bad english.
2- can any help me ? 
I have spent two days with 8 different installations.
Controlnet always leaves me with a black window, or without relation to the image that I place for preprocessing.
And since I installed python 3.09.6 (today) I get some more problems, like this one below.

THANK YOU

Couldn't install sd-webui-controlnet requirement: mediapipe.
Command: "I:\stable-diffusion-webui\venv\Scripts\python.exe" -m pip install mediapipe --prefer-binary
Error code: 1
stdout: Collecting mediapipe
  Using cached mediapipe-0.9.3.0-cp310-cp310-win_amd64.whl (50.3 MB)
Requirement already satisfied: matplotlib in i:\stable-diffusion-webui\venv\lib\site-packages (from mediapipe) (3.7.1)
Requirement already satisfied: attrs>=19.1.0 in i:\stable-diffusion-webui\venv\lib\site-packages (from mediapipe) (23.1.0)
Requirement already satisfied: protobuf<4,>=3.11 in i:\stable-diffusion-webui\venv\lib\site-packages (from mediapipe) (3.20.0)
Collecting sounddevice>=0.4.4
  Using cached sounddevice-0.4.6-py3-none-win_amd64.whl (199 kB)
Requirement already satisfied: flatbuffers>=2.0 in i:\stable-diffusion-webui\venv\lib\site-packages (from mediapipe) (23.3.3)
Requirement already satisfied: absl-py in i:\stable-diffusion-webui\venv\lib\site-packages (from mediapipe) (1.4.0)
Collecting opencv-contrib-python
  Using cached opencv_contrib_python-4.7.0.72-cp37-abi3-win_amd64.whl (44.9 MB)
Requirement already satisfied: numpy in i:\stable-diffusion-webui\venv\lib\site-packages (from mediapipe) (1.23.3)
Collecting CFFI>=1.0
  Using cached cffi-1.15.1-cp310-cp310-win_amd64.whl (179 kB)
Requirement already satisfied: kiwisolver>=1.0.1 in i:\stable-diffusion-webui\venv\lib\site-packages (from matplotlib->mediapipe) (1.4.4)
Requirement already satisfied: contourpy>=1.0.1 in i:\stable-diffusion-webui\venv\lib\site-packages (from matplotlib->mediapipe) (1.0.7)
Requirement already satisfied: python-dateutil>=2.7 in i:\stable-diffusion-webui\venv\lib\site-packages (from matplotlib->mediapipe) (2.8.2)
Requirement already satisfied: cycler>=0.10 in i:\stable-diffusion-webui\venv\lib\site-packages (from matplotlib->mediapipe) (0.11.0)
Requirement already satisfied: pillow>=6.2.0 in i:\stable-diffusion-webui\venv\lib\site-packages (from matplotlib->mediapipe) (9.4.0)
Requirement already satisfied: pyparsing>=2.3.1 in i:\stable-diffusion-webui\venv\lib\site-packages (from matplotlib->mediapipe) (3.0.9)
Requirement already satisfied: fonttools>=4.22.0 in i:\stable-diffusion-webui\venv\lib\site-packages (from matplotlib->mediapipe) (4.39.3)
Requirement already satisfied: packaging>=20.0 in i:\stable-diffusion-webui\venv\lib\site-packages (from matplotlib->mediapipe) (23.1)
Requirement already satisfied: pycparser in i:\stable-diffusion-webui\venv\lib\site-packages (from CFFI>=1.0->sounddevice>=0.4.4->mediapipe) (2.21)
Requirement already satisfied: six>=1.5 in i:\stable-diffusion-webui\venv\lib\site-packages (from python-dateutil>=2.7->matplotlib->mediapipe) (1.16.0)
Installing collected packages: opencv-contrib-python, CFFI, sounddevice, mediapipe

stderr: ERROR: Could not install packages due to an OSError: [WinError 5] Acceso denegado: 'I:\\stable-diffusion-webui\\venv\\Lib\\site-packages\\cv2\\cv2.pyd'
Check the permissions.

[notice] A new release of pip available: 22.2.1 -> 23.1
[notice] To update, run: I:\stable-diffusion-webui\venv\Scripts\python.exe -m pip install --upgrade pip

Warning: Failed to install mediapipe, some preprocessors may not work.

**Comment by dilectiogames** [comment]
did you figured this out?

**Reply by evilfurryone** [reply]
My quick fix was just to delete the 'venv\Lib\site-packages\cv2' folder and rerun the install command, so in this case

"I:\stable-diffusion-webui\venv\Scripts\python.exe" -m pip install mediapipe --prefer-binary

**Reply by soyatlas** [reply]
thanks evifurryone
